pages/interface/components/PublishedSince/index.js

```diff
@@ -1,16 +1,25 @@
-import { formatDistanceToNowStrict } from 'date-fns';
-import { pt } from 'date-fns/locale';
+import { formatDistanceToNowStrict, format } from 'date-fns';
+import { ptBR } from 'date-fns/locale';
+import { Tooltip } from '@primer/react';
 
-export default function PublishedSince({ date }) {
-  return <span suppressHydrationWarning={true}>{formatPublishedSince(date)}</span>;
+function formatPublishedSince(date) {
+  const publishedSince = formatDistanceToNowStrict(new Date(date), {
+    addSuffix: false,
+    includeSeconds: true,
+    locale: ptBR,
+  });
 
-  function formatPublishedSince(date) {
-    const publishedSince = formatDistanceToNowStrict(new Date(date), {
-      addSuffix: false,
-      includeSeconds: true,
-      locale: pt,
-    });
+  return `${publishedSince} atrás`;
+}
 
-    return `${publishedSince} atrás`;
-  }
+function formatTooltipLabel(date) {
+  return format(new Date(date), "EEEE, d 'de' MMMM 'de' yyyy 'às' HH:mm", { locale: ptBR });
+}
+
+export default function PublishedSince({ date }) {
+  return (
+    <Tooltip sx={{ position: 'absolute', ml: 1 }} aria-label={formatTooltipLabel(date)}>
+      <span>{formatPublishedSince(date)}</span>
```

Comment:
```suggestion
      <span suppressHydrationWarning={true}>{formatPublishedSince(date)}</span>
```

Precisa disso para não explodir `Unhandled Runtime Error` com publicação de, por exemplo, poucos segundos atrás.